components/blocks/index.tsx

```
@@ -0,0 +1,197 @@
+    label,
+    dataPath,
+    labelStyle,
+    readOnly,
+    style,
+    dotsType = "normal",
+    hasDescription,
+    ...rest
+  }: DotsBlockType) => {
+    const DotsComponent = dotsType === "3state" ? ThreeStateDots : Dots;
+
+    const [value, setValue] = useSheetDotValue(dataPath!);
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)